test/jira2md.js

```diff
@@ -1,128 +1,227 @@
 var should = require('chai').should();
-var expect = require('chai').expect;
+// var expect = require('chai').expect;
 var fs = require('fs');
 var path = require('path');
 var j2m = require('../index.js');
 
-describe('to_markdown', function() {
-    it('should exist', function() {
-    	should.exist(j2m.to_markdown);
-    });
-    it('should be a function', function() {
-    	j2m.to_markdown.should.be.a('function');
-    });
-    it('should convert bolds properly', function() {
-        var markdown = j2m.to_markdown('*bold*');
-        markdown.should.eql('**bold**');
+describe('to_markdown', function () {
+  it('should exist', function () {
+    should.exist(j2m.to_markdown);
+  });
+
+  it('should be a function', function () {
+    j2m.to_markdown.should.be.a('function');

```

Comment:
Though it's not your changes, do we care about whether subject is a function?

Reply:
I had the same question when I first saw this. I think it doesn't do any harm to leave it, and there's a possibility we could try to export `undefined` and this would break.